src/runtime_src/doc/toc/xrt_ini.rst

```diff
@@ -30,6 +30,23 @@ The following is a simple example that turns on profile timeline trace and sends
    timeline_trace = true
 
 
+**API Support**: From 2020.2 release the runtime configuration options can also be provided through APIs. These APIs are applicable for ``[Runtime]`` group options, not for ``[Debug]`` or ``[Emulation]`` related options.  
```

Comment:
This is not true, sorry misunderstood what you were asking.   The APIs for changing ini values support any group, it's all about the string used as key for the API.

Reply:
Ok, removed that line. Thanks